reference/best_practices/reusable_bundles.rst

```diff
@@ -1,4 +1,164 @@
 Reusable Bundles for Akeneo PIM community
 =========================================
 
-Coming soon!!! :)
+You are an integrator and you want to share reusable parts of your project with the awesome Akeneo PIM community?
+Let's read our pieces of advice to write a great and documented bundle!
+
+
+What is the purpose of a reusable bundle?
+
+* **Compatibility**: to fit Akeneo PIM features
+* **Maintainability**: to make your code flexible and testable (and tested)
+* **Extensibility**: to allow integrators to use your bundle to fit with their specific needs
+* **Scalability**: to know your application limits and document them
```

Comment:
Or maybe something like : "to make your application work with high volume of data and document its behavior in such cases" ?